2016/playbook/session_3/README.md

```diff
@@ -0,0 +1,7 @@
+# Hack Camp session 3
+
+The documentation for this session of Hack Camp is stored in 3 main places.
+
+- [SCHEDULE.md](./SCHEDULE.md): The daily schedule that Hack Camp follows.
+- [ACTIVITIES.md](../ACTIVITIES.md): A detailed listing of activities we run during the camp.
+- [experiments/](./experiments/): A folder of documentation on the experiments we run in camp.
```

Comment:
Yep, you can test the link using a markdown render or your choice (I use  markserv).